src/kili/core/graphql/graphql_client.py

```diff
@@ -58,6 +58,10 @@ def __init__(
             method="POST",
         )
 
+        # Use mutex to avoid multiple processes operating in the cache directory at the same time
+        self._cache_dir_lock = FileLock(
```

Comment:
Does this mean that the lock will be created as soon as the GraphQLCLient will be instantiated?

Reply:
now the lock is active when it's called with .acquire() or .__enter__